frontend/src/libs/enums/article-sub-route.enum.ts

```diff
@@ -1,5 +1,5 @@
 const ArticleSubRoute = {
-  MY_ARTICLES: 'my-articles',
+  MY_ARTICLES: '/articles/my-articles',
```

Comment:
@andanf-e did we decide to make it in a separate route? Not by a filter on all Articles page?

Reply:
It was always a separate route (subroute) 😅